src/calculations.js

```diff
@@ -357,7 +357,20 @@ export const Calculations = {
         this.sound?.playExplosion()
 
         body.starLvl += 1
-        bodies[j].radius = 0n
+
+        if (body.starLvl >= body.maxStarLvl) {
+          // remove from bodies and add to withering bodies
+          bodies.splice(j, 1)
```

Comment:
I think we need to keep in bodies actually because technically the body is still there it just has a radius of 0 so it doesn't impact the other bodies. This is a requirement of how the circuit is written atm : \

Maybe we should rethink when the withering animation actually takes place. In a way it's actually part of the game over animation. The withering could be the "becomes part of the background" since the bodies actually stay in the game after they get hit and don't actually become part of the background until the last frame runs in the chunk of time that the win takes place...

Reply:
yeah could use this as game over by popping all of the bodies into witheringBodies on game over

Reply:
i'll try to switch that now